sdk/java-sdk/src/main/scala/kalix/javasdk/impl/DiscoveryImpl.scala

```diff
@@ -250,10 +250,17 @@ class DiscoveryImpl(system: ActorSystem, services: Map[String, Service], sdkName
 object DiscoveryImpl {
 
   private[impl] def fileDescriptorSet(services: Iterable[Service], userDescPath: String, log: Logger) = {
-    val descriptorsWithSource = loadDescriptorsWithSource(userDescPath, log)
+
+    val descriptors = loadFileDescriptors(userDescPath, log)
```

Comment:
all descriptors we can find in user_function.desc. Doesn't matter if sourceCodeInfo is enabled or not.